addons/sale/views/sale_views.xml

```diff
@@ -675,7 +675,7 @@
                                 <group name="sale_shipping">
                                     <label for="commitment_date" string="Delivery Date"/>
                                     <div name="commitment_date_div" class="o_row">
-                                        <field name="commitment_date"/> <span class="text-muted">Expected: <field name="expected_date" widget="date"/></span>
+                                        <field name="commitment_date"/> <field name="expected_date" widget="date" invisible="1"/>
```

Comment:
You can remove the `widget=date` as it's invisible now.

Can't you remove the `label` and `div` as well? Or are they used somewhere else?

Reply:
You are right for the widget, but I think the label and div should not be deleted since they are used in other modules.
i.e : in sale_order_views.xml --> <xpath expr="//label[@for='commitment_date']" position="before">